libs/wordpress-plugin/plugin/trunk/server/block-db-table.php

```diff
@@ -1,5 +1,24 @@
 <?php
 
+const MINIMUM_MYSQL_VERSION = "8.0.0";
+const MINIMUM_MARIADB_VERSION = "10.2.7";
```

Comment:
Global scope 😁 
```suggestion
const BLOCK_PROTOCOL_MINIMUM_MYSQL_VERSION = "8.0.0";
const BLOCK_PROTOCOL_MINIMUM_MARIADB_VERSION = "10.2.7";
```

Reply:
Addressed in 32165b7a2670623af0c61f7f03c6a6d3778a111e Thank you!